runtime/libpgmath/lib/common/mth_intrinsics.h

```diff
@@ -14,9 +14,18 @@
  * limitations under the License.
  *
  */
+#ifdef __cplusplus
```

Comment:
Sadly, to compile this code with clang++, stdint.h needs to be included instead. I'd suggest something like this:
```
#if defined(__cplusplus) && !defined(__clang__)
#include <cstdint>
#else
#include <stdint.h>
#endif
```